content/development/_index.html

```diff
@@ -52,7 +52,7 @@ <h2>Building &amp; Developing the Haiku Operating System itself</h2>
 <h2 class="icon-document-medium">Legacy Documentation - Not maintained</h2>
 <ul>
 
-<li><strong>Legacy:</strong> <a href="/legacy-docs/bebook/">BeBook</a> | <a href="/legacy-docs/benewsletter/">Be Newsletters</a> | <a href="/legacy-docs/ArtOfBeOSProgramming/">The Art of BeOS Programming</a> (Japanese)</li>
+<li><strong>Legacy:</strong> <a href="/legacy-docs/bebook/">BeBook</a> | <a href="/legacy-docs/benewsletter/">Be Newsletters</a> | <a href="/legacy-docs/ArtOfBeOSProgramming/">The Art of BeOS Programming</a> (Japanese) | <a href="/glass_elevator/rfc/">Requests for Comment</a></li>
```

Comment:
It's a bit sad to put this in "legacy", why not in "development references" above? This will be part of the starting point when we start R2, even if there was not much thinking about it in the last few years, we can review these proposals and add more when it's time for it.

Reply:
Moved!